example/build.gradle

```diff
@@ -0,0 +1,10 @@
+apply plugin: 'java'
+apply plugin: "net.ltgt.apt"
+
+dependencies {
+    compile 'com.google.code.gson:gson:2.6.2'
+    compile 'com.google.auto.value:auto-value:1.2'
```

Comment:
You need both for it to work. Check out the docs here: https://github.com/google/auto/blob/master/value/userguide/index.md#in-pomxml

`apt` provides it to the annotation processing path of `javac`, `compileOnly` puts it in the code completion classpath.